framework/db/mssql/QueryBuilder.php

```
@@ -636,6 +638,51 @@ public function selectExists($rawSql)
+     *
+     * Excludes server-managed `rowversion` columns from the `SET` clause and renders any `rowversion` in the WHERE as a
+     * binary literal, enabling a `rowversion` column to serve as the optimistic lock attribute.
+     *
+     * @see https://github.com/yiisoft/yii2/issues/9653
+     */
+    public function update($table, $columns, $condition, &$params)
+    {
+        $schema = $this->db->getTableSchema($table);
+
+        if ($schema !== null) {
+            foreach ($columns as $name => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)